### ISSUE TITLE
APT source whitelist request for ppa:terry.guo/gcc-arm-embedded

### ISSUE THREAD

**Issue by bherrera** [travis-ci/apt-source-whitelist#132]
https://launchpad.net/~terry.guo/+archive/ubuntu/gcc-arm-embedded

deb http://ppa.launchpad.net/terry.guo/gcc-arm-embedded/ubuntu YOUR_UBUNTU_VERSION_HERE main 
deb-src http://ppa.launchpad.net/terry.guo/gcc-arm-embedded/ubuntu YOUR_UBUNTU_VERSION_HERE main

**Comment by dale-stewart** [comment]
I also need this.  Why is it taking so long to add this one?

**Comment by aethaniel** [comment]
+1, previous issue is https://github.com/travis-ci/apt-source-whitelist/issues/34

**Comment by dale-stewart** [comment]
I found some instructions here that I found useful:
http://blog.travis-ci.com/2015-03-19-uc-berkeley-calsol-team-runs-on-travis-ci/

**Comment by aethaniel** [comment]
Thanks @dale-stewart but I would prefer to have benefits of both containers and cached packages.
Your reference is in fact downloading the whole toolchain source code and compiling it, which isn't a bad idea by itself. 
If we were 100 doing the same thing at the same moment, it would be a total waste of energy.

**Comment by dale-stewart** [comment]
I agree, it is just a stop-gap measure for me until the package becomes available.  However, there is no compilation involved, only downloading the binaries and setting up the path appropriately.  I think the caching could be used to avoid doing the download more than once, but I haven't gotten that far yet.

**Comment by aethaniel** [comment]
@dale-stewart, my apologizes: you were totally right about the avoided compilation: your link shows how to use the precompiled linux32 ARM GCC toolchain in the container context. Ihave read it too fast.

Cheers,
T.

**Comment by dale-stewart** [comment]
No apologies needed, hope it helps!



Sent using CloudMagic Email
[https://cloudmagic.com/k/d/mailapp?ct=pa&cv=7.4.10&pv=5.1&source=email_footer_2] On Sun, Oct 18, 2015 at 4:31 AM, Thibaut VIARD < notifications@github.com [notifications@github.com] > wrote:
@dale-stewart [https://github.com/dale-stewart] , my apologizes: you were totally right about the avoided compilation: your
link shows how to use the precompiled linux32 ARM GCC toolchain in the container
context. Ihave read it too fast.

Cheers,
T.

—
Reply to this email directly or view it on GitHub
[https://github.com/travis-ci/apt-source-whitelist/issues/132#issuecomment-148995081] .[https://github.com/notifications/beacon/AAd5U8C5g1XRL203zBpM1s0onxBZTsYtks5o815TgaJpZM4F4cKg.gif]

**Comment by tve** [comment]
I need this package too to move to the container infrastructure. Seems like no-one at travis-ci is actually listening to these issues, though. Pathetic.

**Comment by kaidokert** [comment]
One more vote. But shouldnt this instead point to : 
https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa

**Comment by kaidokert** [comment]
And i just noticed that as per #390 new apt-sources are not currently added at all :(
